### ISSUE TITLE
RAML directive to set "Inherit auth from parent" in postman after importing schema?

### ISSUE THREAD

**Issue by shafdog** [postmanlabs/raml1-to-postman#80]
In postman, you can set authentication on the collection level (and other levels).  The issue is if a collection is created by importing RAML, each imported endpoint in the collection (created from a RAML schema) has to set the "Inherit auth from parent".

Is there some directive in RAML that cause this to be set?  The securedBy attribute is set to basic & endpoints do show as needing authentication settings.  But it's tedious to set then in a large collection imported from RAML.

Basically I'd like basic auth to be set in **one** place after creating a collection from RAML in API.  I can't figure out the right magic to do it however.

Maybe this is a feature request (or just unfamiliarity with postman ;))?

**Comment by VShingala** [comment]
@shafdog I think RAML directive already defines such things best via `securedBy` which can be used at any resource level. But, this module doesn't have support for using it at collection or folder level yet and all authentication is set at the request level only.

As such, we can treat this as a feature request. We'll update here once we have something concrete on this.
